k8s-cluster/base/caches/secret.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  redis-password: MTRHUmVjczM1dg==
+  redis-password: Tk9UX1RIRV9SRUFMX1BBU1NXT1JE
```

Comment:
Not entirely sure where this came from

Reply:
Came from the helm chart generating the caches. This value isn't used in the ZHE application so a change is inconsequential